sources/core/Stride.Core.AssemblyProcessor/AssemblyProcessorApp.cs

```diff
@@ -214,15 +214,12 @@ public bool Run(ref AssemblyDefinition assemblyDefinition, ref bool readWriteSym
                 if (SerializationAssembly)
                 {
                     processors.Add(new AssemblyScanProcessor());
-                    processors.Add(new SerializationProcessor());
                 }
-
-                if (ModuleInitializer)
+                if (SerializationAssembly)
```

Comment:
Question: This 'if' duplicate was intended right?

Reply:
this PR is just to see the difference to master where it doesnt work